spec/system/support_interface/api_tokens_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.feature 'Manage API tokens' do
+  scenario 'Support creates a token' do
+    given_i_am_signed_in
+    when_i_visit_the_tokens_page
+    and_i_click_on_create_a_token
+    then_i_should_see_a_new_token
```

Comment:
For 110% integration, could the user go on to make a request to `/ping` with the token and see it work?

Reply:
That is a swell idea!

Reply:
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/298/commits/ec9b018a7e80ab32e7e62b6d9ccd0cb52b4a1904

Reply:
💯